config/webpack/webpack.common.js

```diff
@@ -42,6 +42,15 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
         path: path.resolve(__dirname, '../../dist'),
         publicPath: '/',
     },
+    stats: {
+        warningsFilter: [
+            // @react-navigation for web uses the legacy modules (related to react-native-reanimated)
+            // This results in 33 warnings with stack traces that appear during build and each time we make a change
+            // We can't do anything about the warnings, and they only get in the way, so we suppress them
+            './node_modules/@react-navigation/drawer/lib/module/views/legacy/Drawer.js',
```

Comment:
Not sure we should suppress these. Yes they're annoying but if we suppress them we won't remember to upgrade and fix them. We also won't notice when they're gone and thus will never think to remove this code (which will be zombie code by then)

Reply:
> @react-navigation for web uses the legacy modules

Is this likely to change? The logs add a bunch of unactionable junk to the output which makes it more difficult to parse the output. But I get the point about hiding this permanently, so I'm happy to add them back.

Reply:
I'm not sure how such problem are managed here 
On other projects we'll capture an outstanding "to-do" ticket linking to commit/lines of code as a reminder 

I see your point, I can revert the change in my next PR on webpack config. 

Maybe there's a way to log this crap just once, instead for every change made, so many warnings just make a habit of ignoring output altogether...